packages/resources/test/Api.test.ts

```diff
@@ -225,6 +227,28 @@ test("accessLog-props", async () => {
   );
 });
 
+test("accessLog-props", async () => {
```

Comment:
"accessLog-retention" might make more sense for the name.

Reply:
Yes it does. Updated.